### DISCUSSION TITLE
Database Editor

### DISCUSSION THREAD

**Discussion by IssaRevioli** [Ryder25/Ego-Engine-Modding#55] [Q&A] [answered]
Hate to bother ya but is it safe to assume the database editor is outdated?

**Comment by ptasev** [comment, marked as answer]
I'm not completely sure what you mean by outdated.

The database editor was meant to be used for a file called `database.bin` in some of Codemasters' older games. F1 2010-2014, Race Driver: Grid, and Dirt 2 are some of the games that it works for.  With some of the newer games the database.bin file is either hidden from us to prevent modding it, or it simply no longer exists in the case of the games that use erp files. They seem to use a completely different way to store that data now.

**Reply by ptasev** [reply]
Copying from the other comment to complete the answer:
The data in the gamedatabases is much different than the data in the old database.bin files. There's no way to modify gamedatabases as far as I'm aware.

**Comment by IssaRevioli** [comment]
[gamedatabases.zip](https://github.com/ptasev/Ego-Engine-Modding/files/5896798/gamedatabases.zip)
So this is not the same kind of file?

**Reply by ptasev** [reply]
Right. This data in the gamedatabases is much different than the data in the old database.bin files. There's no way to modify currently as far as I'm aware.

**Reply by IssaRevioli** [reply]
Man codies are not making it easy but thank you anyway.
